docker-based-rock-ons/overview.rst

```diff
@@ -76,6 +76,19 @@ If no Rock-ons are showing on the **All** tab then click the **Update** button
 to refresh the list of available Rock-ons. To install a listed Rock-on use
 its **Install** button on the Rock-ons WebUI page.
 
+.. _user_group:
```

Comment:
I would prefer further specification to differentiate it from our Users/Groups dedicated page:
```suggestion
.. _rockons_user_group:
```